src/sentry/integrations/slack/actions/notification.py

```diff
@@ -101,8 +104,10 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
                 "unfurl_links": False,
                 "unfurl_media": False,
             }
+            json_blocks = None
             if payload_blocks := blocks.get("blocks"):
-                payload["blocks"] = orjson.dumps(payload_blocks).decode()
+                json_blocks = orjson.dumps(payload_blocks).decode()
```

Comment:
Is there a test covering this line of code? After #inc-779 I get scared whenever I see `orjson.dumps`

Reply:
i added testing to this area of the code in https://github.com/getsentry/sentry/pull/72066, it should be covered now